jdaviz/configs/default/aida.py

```diff
@@ -173,6 +203,17 @@ def _get_current_center(self, sky_or_pixel, image_label=None):
 
         return center
 
+    def _get_current_rotation(self, sky_or_pixel):
+        orientation = self.app._jdaviz_helper.plugins.get('Orientation', None)
+        # rotation angle from pixel y
+        rotation_angle = orientation.rotation_angle
+
+        # rotation angle of pixel y from north
+        degn = orientation._obj._get_wcs_angles()[-3]
+
+        rotation = rotation_angle - degn
+        return rotation * u.deg if (sky_or_pixel in ("sky", None)) else rotation
```

Comment:
Let's always return rotation as a Quantity. A angular units have meaning in pixel coords too.